pangeo_forge_orchestrator/models.py

```diff
@@ -1,10 +1,63 @@
 from datetime import datetime
 from enum import Enum
-from typing import Optional
+from typing import List, Optional
 
-from sqlmodel import SQLModel
+from sqlmodel import Field, SQLModel
 
-from .model_builders import MultipleModels
+from .model_builders import MultipleModels, RelationBuilder
+
+# Bakery --------------------------------------------------------------------------------
+
+
+class BakeryBase(SQLModel):
+    """Information about a Pangeo Forge Bakery.
+
+    :param region: The cloud region.
+    :param name: The bakery name.
+    :param description: A description of this bakery.
+    """
+
+    region: str  # TODO: Categorical constraint.
+    name: str  # TODO: Unique constraint.
+    description: str
+
+
+class BakeryRead(BakeryBase):
+    """The bakery read model. See ``RecipeRunRead`` docstring in this module for further detail.
+    """
+
+    id: int
+
+
+# Feedstock -----------------------------------------------------------------------------
+
+
+class GitHubOrgs(str, Enum):
+    """Categorical choices for ``FeedstockBase.github_org``."""
+
+    pangeo_forge = "pangeo-forge"
+
+
+class FeedstockBase(SQLModel):
+    """Information about a Pangeo Forge Feedstock.
+
+    :param github_repo: The name of the feedstock repo, e.g. ``"noaa-oisst-avhrr-only-feedstock"``.
+    :param github_org: The name of the GitHub org in which this feedstock repo resides. Must be one
+      of the options defined in ``GitHubOrgs``.
+    """
+
+    github_repo: str
+    github_org: GitHubOrgs = GitHubOrgs.pangeo_forge
```

Comment:
Sorry to flip flop on this so much. But I've really been thinking about the best way to refer to feedstocks. The way this is currently done (based on my prior comment) explicitly assumes github is the provider. It might be nice to future-proof ourselves around the idea of multiple possible providers (e.g. we may one day want to integrate with gitlab).

A great template for how this is done is binder:
- Documentation of different RepoProviders: https://binderhub.readthedocs.io/en/latest/reference/repoproviders.html#zenodoprovider
- Code: https://github.com/jupyterhub/binderhub/blob/master/binderhub/repoproviders.py
- Resolver: https://github.com/jupyterhub/binderhub/blob/c08d5dbadccd5fbd831b8574570bec3219fb4d15/binderhub/main.py#L15-L24

So bottom line, I would recommend changing this to

```suggestion
    provider: RepoProvider = RepoProvider.github
    spec: str
```

where spec is dependent on the provider. For github, it would be a string like `pangeo-forge/my-great-recipe`. (I would not assume that all repos will live in the pangeo-forge org).